ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -145,7 +145,12 @@ export class ImportedDocuments extends React.Component {
       data?.entries_count;
 
     if (error) {
-      return <NotFound title="Task not found" />;
+      const {
+        response: {
+          data: { message, status },
+        },
+      } = error;
+      return <NotFound title={status} message={message} isBackOffice />;
```

Comment:
isn't the name of the component misleading ? normally I would expect notFound = 404 only